### PR TITLE
D3 Visualizer

### PR DESCRIPTION
**Pull request by TRnonodename**
This is a pull request to include the D3 visualizer in the docs/examples directory into the main trunk. I've included a reference to it in the documentation for AddFileInfoton but I wonder, @nruppin whether it makes sense to include an examples part of the documentation?

### CODE REVIEW

**Comment by TRnonodename** [comment]
OK.. working too fast this morning.. I've also added a initial contributions policy to the root of the project. @EmmanuelCanaan  would appreciate your eyes on that.

File: docs/examples/visualizer/index.html [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+<!-- Debug with python -m SimpleHTTPServer 8000 -->
+<!-- Post to CM-Well with 
+    - curl -X POST <env>/exercise/files/visualizer.html -H "X-CM-WELL-TYPE: FILE" -H "Content-Type: text/html" -ndex.html"  
+        for the main file and
+    - curl -X POST <env>/exercise/files/d3.min.js -H "X-CM-WELL-TYPE: FILE" -H "Content-Type: application/javascript" --data-binary @"d3.min.js" 
+        for d3
+-->
+
+<head>
+    <title>Stick and ball</title>
```
Comment (bryaakov) [inline docs/examples/visualizer/index.html]: Title should be more meaningful, e.g. "CM-Well Data Visualizer App POC"

File: docs/examples/visualizer/index.html [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+                return_uri += "?format=jsonl";
+            }
+            return return_uri;
+        }
+
+        /**
+            Don't render system infotons
+        **/
+        function isSystemInfoton(uri) {
+            if (String(uri).endsWith(".sys"))
```
Comment (bryaakov) [inline docs/examples/visualizer/index.html]: Sorry for being petty, but please refactor to `return String(uri).endsWith(".sys"))` (no need for an `if` here)

File: docs/examples/visualizer/index.html (hunk 1 above)
Comment (TRnonodename) [reply docs/examples/visualizer/index.html]: Done.

File: docs/examples/visualizer/index.html (hunk 2 above)
Comment (TRnonodename) [reply docs/examples/visualizer/index.html]: Duh. I'm embarrassed by this. Thanks for keeping me honest.

**Pull request merged**